src/Azure/AzureAD/Authentication.AzureAD.UI/src/Areas/AzureAD/Controllers/AccountController.cs

```diff
@@ -13,17 +13,23 @@ namespace Microsoft.AspNetCore.Authentication.AzureAD.UI.AzureAD.Controllers.Int
     [Route("[area]/[controller]/[action]")]
     internal class AccountController : Controller
     {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
why? I don't think that the Microsoft.Identity.Web templates use them?

Reply:
Exactly, these are the old APIs that we used in our templates. After converting to using Microsoft.Identity.Web, we are deprecating the old APIs.

Reply:
Just to clarify, everything in Auth.AzureAD.UI and Auth.AzureADB2C.UI are deprecated. We're not removing it since we have a policy of marking everything as deprecated for 1 release before removing them.